libs-sol/nearbridge/contracts/NearBridge.sol

```diff
@@ -212,7 +212,7 @@ contract NearBridge is INearBridge {
         for (uint i = 0; i < backup.next_bps_length; i++) {
             backup.next_bps[i] = last.next_bps[i];
         }
-        for (uint i = 0; i < backup.approvals_after_next.length; i++) {
```

Comment:
In solidity why we would also keep a var approvals_after_next_length? backup.approvals_after_next.length seems to work

Reply:
It does not compile.

Reply:
Got it